### ISSUE TITLE
Some question about similarity_combo?

### ISSUE THREAD

**Issue by xuzhongyou** [msracver/Deep-Exemplar-based-Colorization#11]
Thank u for your job. it helps a lot. I have ran the Deep-Image-Analogy on linux sucessfully. but i met some problems when do as your advises like 

@TruePhone The Similarity SubNet (and Deep Image Analogy if you use it to build correspondence) is implemented in C++. It should be easy to transplant to Linux. Deep Image Analogy already has a Linux version (https://github.com/msracver/Deep-Image-Analogy/tree/linux) and our Similarity SubNet is actually built based on it. You may copy the source code from Deep-Exemplar-based-Colorization/similarity_subnet/windows/similarity_combo/ to msracver/Deep-Image-Analogy/tree/linux and slightly modify it on Linux.

but when i compile, it missing some files.h like windows.h . sorry, i am not familiar with c++ in windows. can u give me some advises to get similarity_combo file in linux?

**Comment by hmmlillian** [comment]
@xuzhongyou Yes, that's a problem. We have to do some modification before compiling the code on Linux. We will do it soon. Thanks for your suggestion.

**Comment by mattclifford1** [comment]
Did you ever get round to getting this working in Linux

**Comment by xuzhongyou** [comment]
not yet

**Comment by xuzhongyou** [comment]
> Did you ever get round to getting this working in Linux

not yet
